index.js

```diff
@@ -93,13 +94,15 @@ function _javascript (entry, opts, setCss) {
     cache: {}
   }
 
-  opts = xtend(base, opts || {})
+  const jsOpts = xtend(base, opts.js || {})
 
-  const b = (this.optimize)
-    ? browserify(opts)
-    : watchify(browserify(opts))
-  b.plugin(cssExtract, { out: createCssStream })
-  b.transform(sheetify)
+  const b = opts.optimize
+    ? browserify(jsOpts)
+    : watchify(browserify(jsOpts))
+  if (!opts.cssDisabled) {
```

Comment:
Not sure if this is the correct way to implement `cssDisabled`, but I don't have any reference to go by.